packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -1,219 +1,393 @@
-import { contains, fromHtml, isNode, isRtl, VTable } from 'roosterjs-editor-dom';
```

Comment:
Can you double check your whitespace/lineendings on this (and other files) in your PR? We shouldn't be seeing deltas like this.

Reply:
I have prettier enabled in VSCode by default, so depending on how that's setup it's probably doing this